styles/styles.css

```diff
@@ -0,0 +1,91 @@
+/*
+Colors:
+
+orange: #F5C85F
+light-grey: #EEE
+gray: #999
+dark-grey: #111
+
+Font: Open Sans
+*/
+
+body {
```

Comment:
i'm leaving only this in the body style since i'm using the same font everywhere, is this ok or should i create a new class and use it in all my big divs?  

Also i added the padding and margin because there was a small margin at the left of the body that i hadn't seen.

body {
	font-family: "Open Sans", sans-serif;
	padding: 0;
	margin: 0;
}

Reply:
Use the equalizer I mentioned and it will reset all those margins and paddings. You don't have to create a class for the font, do as I commented below for the fonts and it should be OK :smile:

Reply:
sorry i didnt see the equalizer, though it breaks some of the other stuff that i made. Should i use this in every html? because i want some of the default behavior (im using h3 and h2 for example), plus for some reason it still doesnt remove the margin on the left, i still have to do it manually modifying the body tag style.

Reply:
Not all browsers apply the same sizes or styles to the tags like h1, h2, etc, so it's a good practice to add a normalizer so you can follow the original design without any problems.It's really strange you need to add thos margins to the body, maybe there's some class that's changing it :thinking: